src/main/java/io/github/patternatlas/api/entities/user/role/PrivilegeConstant.java

```diff
@@ -0,0 +1,72 @@
+package io.github.patternatlas.api.entities.user.role;
+
+import org.springframework.stereotype.Component;
+
+@Component("PC")
```

Comment:
The reasoning was to have a short name for it since it is used in every PreAuthorize check as for example 
``@PostFilter("hasResourcePermission(filterObject.id, @PC.ISSUE_READ)")`` and those annotations get large pretty fast and a longer name might make the actual privilege name (i.e. ISSUE_READ) harder to grasp quickly. Should I still change it?